index.rst

```diff
@@ -1,61 +1,114 @@
-..
-  Technote content.
+:tocdepth: 1
 
-  See https://developer.lsst.io/restructuredtext/style.html
-  for a guide to reStructuredText writing.
+.. sectnum::
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+Abstract
+========
 
-  Use the following syntax for sections:
+Butler manages read and write access to Rubin Observatory data.
+Different types of data require different access control rules (Data Release versus User Generated data products, for example).
+Butler itself also needs a backing SQL database to store its information, which should be protected from unauthorized modification.
+This tech note proposes a design for access control for both Butler queries and for data upload and download mediated by Butler.
 
-  Sections
-  ========
+Current state
+=============
 
-  and
+Butler is a Python library that mediates access to Rubin Observatory data.
+It uses a SQL database to store metadata about files.
+That database can either be a local SQLite database or a remote SQL database.
 
-  Subsections
-  -----------
+When a Rubin Science Platform user wants to process some data, they would call Butler to download the data, perform whatever processing they wish, and then call Butler again to store the results somewhere for future use.
+This requires the user running Butler to have direct access to the backing SQL database used by Butler.
+This database is called the registry.
+For Data Release data products, the Butler registry will be a central SQL database.
 
-  and
+Users are expected to want to perform Butler queries that combine results from a Data Release with results from User Generated data products (their own or those of collaborators).
+Butler uses a single database for both types of data products so that it can perform these sorts of queries easily.
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+Therefore, to run Butler, users must have credentials with access to the database for the Butler registry.
+To store or retrieve files, users must also have credentials with access to those files.
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+Constraints
+===========
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
+Requirements
+------------
 
-     Caption text.
+#. Users must be able to query the combination of Data Release and User Generated data products using Butler.
+#. Users must be able to download data products to which they have access, including the Data Release, and upload their own User Generated data products.
+#. Users must not have access to User Generated data products or the registry entries for those products unless they have been granted access by the owners of those User Generated data products.
```

Comment:
Append: ", either directly or via membership in a group that has been granted access." to tie this together with the A&A requirements.